properties/service_fabric_managed_netframework.targets

```diff
@@ -50,6 +51,8 @@
   <!-- Copy prod  binaries for full net framework-->
   <Target Name="CopyProdBinaries_netframework">
     <ItemGroup>
+	    <BinairesNetFramework Include="src\Microsoft.ServiceFabric.StartupServicesUtility\$(OutputPath)WindowsFabricServiceModel\StartupServicesModel.xsd"/>
```

Comment:
Rename WindowsFabricServiceModel -> StartupServicesModel, as this model has nothing to do with WinfabModelServiceModel

Reply:
Fixed.